src/styles/examples.css.js

```diff
@@ -39,6 +36,7 @@ aside.example div.example {
   border-color: #999;
   background: #fff;
 }
+
 aside.example div.example span.example-title {
```

Comment:
I feel like there should've been some commas here.. 🤔

Reply:
yeah, that's weird.... maybe we can remove this entirely.